### ISSUE TITLE
Update the notes of GlassBR's DDs to properly reference other models and definitions

### ISSUE THREAD

**Issue by niazim3** [JacquesCarette/Drasil#780]
After #779's branch is merged into master, the output will match the expected one documented in issue #350.

Once `Label` is stable enough for reference implementation, can you please update the sentences in the descriptions of the data definitions to link to the appropriate DD, IM, or TM?

Example of what needs to be updated below:
![image](https://user-images.githubusercontent.com/28247301/42392182-fca5ae92-811f-11e8-8b57-3d60efa42c99.png)
